sdk/schemaregistry/azure-schemaregistry/tests/async_tests/test_schema_registry_async.py

```diff
@@ -17,24 +17,42 @@
 # IN THE SOFTWARE.
 #
 # --------------------------------------------------------------------------
+import os
 import functools
 import pytest
 import json
+import sys
 
 from azure.schemaregistry.aio import SchemaRegistryClient
+from azure.schemaregistry import ApiVersion
 from azure.identity.aio import ClientSecretCredential
 from azure.core.exceptions import ClientAuthenticationError, ServiceRequestError, HttpResponseError
 
 from devtools_testutils import AzureRecordedTestCase, EnvironmentVariableLoader
 from devtools_testutils.aio import recorded_by_proxy_async
 
+# TODO: add protobuf env var for live testing when protobuf changes have been rolled out
+is_livetest = str(os.getenv("AZURE_TEST_RUN_LIVE")).lower()
+if is_livetest == "true":   # protobuf changes have not been rolled out
+    sr_namespaces = {
```

Comment:
nit you can define your default dict and just update it with the values needed for protobuf, since that seems to be the only difference here.

Reply:
so I tried doing that but it was failing for some reason: https://github.com/Azure/azure-sdk-for-python/pull/32341/commits/3f76779a501424589eba055fd8f11fb112df8850

Reply:
updated to use `is_livetest != "true"` rather than `is_livetest == "false"` since maybe that var is set to None. will see if that works.